sql/mob_pool_mods.sql

```diff
@@ -705,6 +705,12 @@ INSERT INTO `mob_pool_mods` VALUES (5922,6,150,0);      -- MPP: 150
 INSERT INTO `mob_pool_mods` VALUES (5944,3,-10,0);      -- HPP: -10
 INSERT INTO `mob_pool_mods` VALUES (5944,6,35,0);       -- MPP: 35
 
+-- Trust: Prishe II
```

Comment:
I don't remember what MNKs get, but are these in addition to what she'll get by being marked as MNK/WHM in mob_pools?

Reply:
the HP and MP mods were added based on testing.
I had a naked level 99 summon her and compared her HP and MP with a naked level 99 on LSB. Added mods until we about matched. What prompted me to even look at this was this comment on BGWiki
![image](https://github.com/LandSandBoat/server/assets/65316697/2bcc8497-24a1-4bc1-88aa-2ea6800458ef)

The double attack is set to 100% because otherwise she will swing with 1 fist per attack round

Reply:
Fix her weapon in her weapon instead, shes 1 hand punching because shes not set to h2h for her weapon in her pool data

Reply:
fixed weapon in mob pools, removed mods